python/tests/eager/test_library_kernels.py

```diff
@@ -0,0 +1,69 @@
+# ============================================================================ #
```

Comment:
Can you run the whole Python test suite with same named test files? I recall that since we don't have `__init.py__` files in the individual test directories, they need to be uniquely named. So, I've renamed those under 'eager' with suffix `_eager`

Reply:
I didn't know that - I'll update the file name.